docs/internal_api.yml

```diff
@@ -7,9 +7,9 @@ info:
     Not exposed via the API Gateway - intended for internal use only.
 
 basePath: '/api/internal/v1/inventory'
-host: 'mender-inventory:8080'
+host: 'hosted.mender.io'
```

Comment:
This API is not exposed on hosted mender, and it is considered internal.
I don'w know what the policy is for these end-points, but hosted.mender.io is wrong.

Reply:
Thanks. Fixed now.